test_unstructured/partition/test_text.py

```diff
@@ -18,13 +18,22 @@
 ]
 
 
-def test_partition_text_from_filename():
-    filename = os.path.join(DIRECTORY, "..", "..", "example-docs", "fake-text.txt")
-    elements = partition_text(filename=filename)
+@pytest.mark.parametrize("filename,encoding", [('fake-text.txt', 'utf-8'), ('fake-text.txt', None), ('fake-text-utf-16-be.txt', 'utf-16-be')])
+def test_partition_text_from_filename(filename, encoding):
+    filename = os.path.join(DIRECTORY, "..", "..", "example-docs", filename)
+    elements = partition_text(filename=filename, encoding=encoding)
     assert len(elements) > 0
+    print([str(e) for e in elements])
```

Comment:
```suggestion
```
Perhaps this was leftover from debugging? I think it's best to remove these kinds of`print` calls from tests.

Reply:
Good catch, yes definitely left over from debugging :)